reconcile/terraform_tgw_attachments.py

```diff
@@ -1,157 +1,213 @@
 import json
 import logging
 import sys
-from typing import Any
+from typing import (
+    Any,
+    Callable,
```

Comment:
We import `Mapping` and `Callable` from now on from `collections.abc` for [PEP585](https://peps.python.org/pep-0585/) compliance.

Reply:
updated